### DISCUSSION TITLE
OSPO Definition 📗 Second release is out!

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#333] [Announcements]
We are proud to announce that [the OSPO Definition v. 2.0 release is live](https://ospoglossary.todogroup.org/ospo-definition/).

* 📝 [Release notes can be found here](https://github.com/todogroup/ospodefinition.org/releases/tag/v1.2.0)
* 📖 [OSPO Definition (repo) can be found here](https://github.com/todogroup/ospodefinition.org#open-source-program-office-ospo-definition-and-guide)

💚 Big thanks to all the contributors who reviewed and helped in making the new release possible, including:

* Cornelius Schumacher
* Shanmugapriya Manoharan
* Ulises Gascón
* Georg Kunz
* Russell Rutledge
* Sourav Das
* Chris Yang
* Chris Lloyd-Jones
* Chris Aniszczyk
* Leslie Hawthorn
* Ana Jimenez Santamaria
* Katrina Novakovic
* Jonas van den Bogaard

+ all the contributors participating in the OSPO Book project, who laid the foundation for this new version as part of the [OSPO book initiative](https://todogroup.org/blog/ospo-book-project/) 💡

👋 Feedback and improvements are always welcome! check our [contributing guide to get started.](https://github.com/todogroup/ospodefinition.org/blob/master/CONTRIBUTING.md)

**Comment by node** [comment]
Cool. It is worth of translating into more languages.

**Comment by shu-mutou** [comment]
Hi, I'd like to translate this into Japanese. Is there a specific method for creating translations? Should I create a README-ja.md and translate it? Thanks!

**Comment by Nliver** [comment]
Cool~
